digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala

```diff
@@ -1899,6 +1899,58 @@ Left|      |Right
     }
   }
 
+  test("Test checkProjectElyCodes When discontinuity is 3 on one track but not on the other of a road part Then validator should return errors") {
```

Comment:
test("Test checkProjectElyCodes When discontinuity is 3 (ELY change) on one track but not on the opposite track Then validator should return errors")